src/pages/Options/index.css

```diff
@@ -2,8 +2,9 @@ body {
   margin: 0;
   padding: 0;
   color: #ffffff;
-  background-color: #0C0C0C;
+  background-color: #0c0c0c;
   position: relative;
+  overflow-x: hidden;
```

Comment:
I'm curious what did this solve for?

Reply:
[scrnli_1_11_2024_2-24-51 PM.webm](https://github.com/secretkeylabs/xverse-web-extension/assets/65149726/f9170b5c-e65f-49b5-8d74-ec3d86da5c7f)

`overflow-x: hidden;` removes the horizontal scrollbar here